action.yml

```diff
@@ -28,7 +28,7 @@ runs:
         python-version: "3.8"
 
     - name: install prefect
-      run: pip install 'prefect~=2.8.7' 'pydantic>=1.10.0,<2.0.0' 'anyio<4' 'apprise<1.8.0' 'typer~=0.9.4'
+      run: pip install 'prefect~=2.8.7' 'pydantic>=1.10.0,<2.0.0' 'anyio<4' 'apprise<1.8.0' 'typer~=0.9.4' 'griffe<1.0.0'
```

Comment:
as this is just the prefect cli, do we even need to stick with 2.8, or could we go to 2.20 where I believe they've resolved dependency issues like this already?

Reply:
Good point - I'll give a Prefect bump a go 👍